src/app/cfg.rs

```diff
@@ -1,34 +1,43 @@
-use super::types::{SinkFn, UpdateFn, ViewFn, WindowEventsFn};
-use super::{builder::IntoAfterMount, MountType};
+use super::types::{UpdateFn, ViewFn};
 use crate::virtual_dom::IntoNodes;
-use std::marker::PhantomData;
 use std::rc::Rc;
 
 #[allow(clippy::module_name_repetitions)]
-pub struct AppInitCfg<Ms, Mdl, INodes, GMs, IAM: ?Sized>
+pub struct AppCfg<Ms, Mdl, INodes>
 where
     Ms: 'static,
     Mdl: 'static,
     INodes: IntoNodes<Ms>,
-    IAM: IntoAfterMount<Ms, Mdl, INodes, GMs>,
 {
-    pub mount_type: MountType,
-    pub into_after_mount: Box<IAM>,
-    pub phantom: PhantomData<(Ms, Mdl, INodes, GMs)>,
+    pub(crate) document: web_sys::Document,
+    // @TODO_B: get more context of use in order to understand how to remove
```

Comment:
Might need help on this one

Reply:
I think `mount_point` it's required only in `App::start` body now but I can be wrong.

It was needed when we had two configs to remember `mount_point` during building the `App` with `Builder` or with the oldest start API builder pattern.

Reply:
Not sure what `patch_els` does, but it also uses the `mount_point`. I think it's needed, but you might be able to remove it from there as well.